charts/forms-flow-ai/templates/configmap.yaml

```diff
@@ -17,27 +17,27 @@ data:
   KEYCLOAK_URL: https://{{ tpl (index .Values "forms-flow-idm" "keycloak" "ingress" "hostname") . }}
   KEYCLOAK_URL_REALM: {{ index .Values "forms-flow-idm" "realm" }}
-  FORMIO_URL: https://forms-flow-forms-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
-  FORMIO_DOMAIN: https://forms-flow-forms-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
-  WEB_API_URL: https://forms-flow-api-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
+  FORMIO_URL: https://{{tpl (index .Values "forms-flow-forms" "ingress" "hostname") .}}
+  FORMIO_DOMAIN: https://{{tpl (index .Values "forms-flow-forms" "ingress" "hostname") .}}
+  WEB_API_URL: https://{{tpl (index .Values "forms-flow-api" "ingress" "hostname") .}}
   MONGODB_URI:  {{ ternary (tpl "mongodb://{{ .Values.mongodb.auth.usernames | first }}:{{ .Values.mongodb.auth.passwords | first }}@{{ .Values.mongodb.service.nameOverride }}:{{ .Values.mongodb.service.ports.mongodb }}/{{ .Values.mongodb.auth.databases | first }}" .) "<DEFINE_ME>" .Values.mongodb.enabled | quote }} 
   {{- if .Values.mongodb.enabled }} 
   NODE_CONFIG: '{"mongo":"mongodb://{{ .Values.mongodb.auth.usernames | first }}:{{ .Values.mongodb.auth.passwords | first }}@{{ .Values.mongodb.service.nameOverride }}:{{ .Values.mongodb.service.ports.mongodb }}/{{ .Values.mongodb.auth.databases | first }}"}'
   {{- else }}
   NODE_CONFIG: "<DEFINE_ME>"
   {{- end }}
-  FORMSFLOW_WEB_URL: https://forms-flow-web-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
-  FORMSFLOW_ADMIN_URL: https://forms-flow-admin-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}/api/v1
-  FORMSFLOW_ADMIN_BASE: https://forms-flow-admin-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}/api
+  FORMSFLOW_WEB_URL: https://{{tpl (index .Values "forms-flow-web" "ingress" "hostname") .}}
+  FORMSFLOW_ADMIN_URL: https://{{tpl (index .Values "forms-flow-admin" "ingress" "hostname") .}}/api/v1
+  FORMSFLOW_ADMIN_BASE: https://{{tpl (index .Values "forms-flow-admin" "ingress" "hostname") .}}/api
   MODEL_ID: "Seethal/sentiment_analysis_generic_dataset"
-  INSIGHT_API_URL: https://forms-flow-analytics-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
+  INSIGHT_API_URL: https://{{tpl (index .Values "forms-flow-analytics" "ingress" "hostname") .}}
   KEYCLOAK_ENABLE_CLIENT_AUTH: "false"
   MULTI_TENANCY_ENABLED: "false"
-  DATA_ANALYSIS_URL:  https://forms-flow-data-analysis-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
+  DATA_ANALYSIS_URL:  https://{{tpl (index .Values "forms-flow-data-analysis" "ingress" "hostname") .}}
   NODE_ENV: "production"
   CUSTOM_SUBMISSION_URL: ""
   CUSTOM_SUBMISSION_ENABLED: "false"
-  FORMSFLOW_DOC_API_URL: https://forms-flow-documents-api-{{.Release.Namespace}}.{{tpl (.Values.Domain) .}}
+  FORMSFLOW_DOC_API_URL: https://{{tpl (index .Values "forms-flow-documents-api" "ingress" "hostname") .}}
```

Comment:
we can follow with the existing line of code.

Reply:
With the existing line, it is not possible to configure the domain separately, such that e.g. the namespace is not used in the domain. 

Due to the long subdomains (especially with the namespace, that is not required) it is quite easy to run into the character limit for the subject alternative name in tls certificates.